src/components/StyledImageContainer.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+
+const StyledImageContainer = styled.div`
+  background-color: ghostwhite;
+  border-radius: 20px;
+  margin-bottom: 20px;
+  position: relative;
+  margin-top: 50px;
+  border: 0px solid orange;
+  filter: drop-shadow(0px 0px 3px white);
```

Comment:
add width based on image width. wrap title if it's too long, the favorite button is getting outa bounce here.

![attachement](https://cdn.discordapp.com/attachments/549037120522682368/1088368552467234856/image.png)